05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
 - **Browser compatibility** solutions so there is consistent style across browsers
 - Creation of standard **CSS classes** which can be used to style advanced components of the user interface
 
-The most popular frameworks are Bootstrap and Foundation. Here's [an article](https://www.upwork.com/hiring/development/bootstrap-vs-foundation-which-framework-is-right-for-you/) comparing the two Frameworks. We'll be digging into Foundation, but the concepts apply to both frameworks.
+The most popular frameworks are Bootstrap and Foundation. Here's [an article](https://www.upwork.com/hiring/development/bootstrap-vs-foundation-which-framework-is-right-for-you/) comparing the two Frameworks. We'll be digging into Bootstrap, but the concepts apply to both frameworks.
```

Comment:
This article has become way more... uh... trying to sell a product. Is there a way we can find a different resource? Also if anything, I think it'd be fine to just list "The most popular CSS frameworks are"

Also, my hot take on this list should be:
- Bootstrap
- Foundation
- Material Design
- uikit
- Semantic UI